package/src/renderer/__tests__/e2e/Colors.spec.tsx

```diff
@@ -0,0 +1,38 @@
+import { surface } from "../setup";
+
+describe("Colors", () => {
+  it("should support int, string, array, and Float32Array", async () => {
+    const result = await surface.eval((Skia) => {
+      const areEqualFloat32Arrays = (...arrays: Float32Array[]) => {
+        // Check if all arrays have the same length
+        const allSameLength = arrays.every(
+          (array) => array.length === arrays[0].length
+        );
+        if (!allSameLength) {
+          return false;
+        }
+
+        // Compare elements across all arrays for each index
+        for (let i = 0; i < arrays[0].length; i++) {
+          if (!arrays.every((array) => array[i] === arrays[0][i])) {
+            return false;
+          }
+        }
+
+        return true;
+      };
+
+      const c1 = Skia.Color("cyan");
+      const c2 = Skia.Color([0, 1, 1, 1]);
+      const c3 = Skia.Color(0xff00ffff);
+      const c4 = Skia.Color(Float32Array.of(0, 1, 1, 1));
```

Comment:
Hey, thanks! I should have added these myself.

I assumed from [#2113][0] that `number[]` expected 0-255 (and so coded it that way on the C++ side). Should I change it to 0-1 to match Float32Array?

[0]: https://github.com/Shopify/react-native-skia/issues/2113

Reply:
I actually not sure I need to check 🙈 also a good test to add would be to use the drawOffscreen API to make sure they all generate the same image (cyan).
I tried to write more info at https://github.com/Shopify/react-native-skia/pull/2145/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5